src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -241,10 +241,37 @@ public static void main(String[] args) {
 
   /**
    * If one file defines a name and another uses it as a namespace, we have the
-   * Constants.COLLDING_PROVIDE_ALIAS_POSTFIX workaround. In partial mode, Clutz can't see all
+   * Constants.COLLIDING_PROVIDE_ALIAS_POSTFIX workaround. In partial mode, Clutz can't see all
    * definitions of a name, so the list of names that require aliases must be passed as an input.
+   * Also Closure Library uses some names as both classes and namespases. For example,
+   * `goog.ui.Component` is not only a class but also a namespace in `goog.ui.Component.EventType`.
```

Comment:
Can you add a link to the github repo containing goog.ui, so when one reads the comment they can quickly click and see the example at the source.

Reply:
added the link